poshc2/client/Help.py

```diff
@@ -515,6 +515,13 @@
 nuke-autorun
 automigrate-frompowershell (alias: am)
 
+Hosted-Files:
+====================
+show-hosted-files
+add-hosted-file
+del-hosted-file
```

Comment:
Is this better as `disable-hosted-file`? More accurate I think and matches `enable-hosted-file`.